nextflow.config

```
@@ -62,6 +62,12 @@ params {
     // General module configuration
     multiqc.config         = "$projectDir/configs/multiqc_conf.yaml"
     multiqc_assembly_report_config = "$projectDir/configs/multiqc_assembly_report_config.yml"
+
+    // define HiC mapping pipeline [ "bwa", "bwa-mem2", "minimap2", "strobealign", "" ]
+    hic_mapper             = 'bwa-mem2' 
+    // define HiC workflow strategy [ "arima", "omni-c", "sanger" ]
+    hic_pipeline           = 'arima'
```

```suggestion
    hic_alignment_protocol           = 'arima'
```
and then here I would use alignment_protocol rather than pipeline. This I think will be important to name correctly so users aren't confused since `pipeline` is a general unspecific term. Using protocol, we should be able to point to a doc somewhere or code at worst.